unrarall

```diff
@@ -165,6 +167,40 @@ function getUnrarFlags()
   esac
 }
 
+#function to check if an archive have been extracted already
+function isAlreadyExtracted()
+{
+  case "$1" in
+    unrar)
+      contains=($(unrar lb "$2"))
+      for x in $contains; do
```

Comment:
Hey!

Yeah sorry, indentation is probably incorrect, used my Macbook (which i do not use for coding normally) while kids was sleeping (was on parental leave)  ;)
Please feel free to change anything :)